index.js

```diff
@@ -1,25 +1,28 @@
 const express = require('express'),
       request = require('request'),
+      pug = require('pug'),
       morgan = require('morgan');
 
 var app = express();
 
+app.set('view engine', 'pug');
```

Comment:
answer: It allows us to use response.render()! When we set a view engine like this to an app then we dont need to put the extension in the path. Also, res.render() will lookup templates under /views directory by default.